http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -758,10 +760,7 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
   private[service] def getSavableCluster(
     userEmail: WorkbenchEmail,
     cloudContext: CloudContext,
-    now: Instant,
-    numNodepools: Option[NumNodepools],
```

Comment:
`numNodepools` and `clusterName` were never specified to this method, so removed them.

(I think this was a holdover to when we have a batch-nodepool-creation endpoint, which was removed.)